cfgov/jinja2/v1/_includes/molecules/archived-pill.html

```diff
@@ -1,3 +1,23 @@
-{% if page and page.is_archived %}
-	<div class="m-archived-pill">Archived</div>
-{% endif %}
\ No newline at end of file
+{# =========================================================================
+
+   archived_pill.render()
+
+   =========================================================================
+
+   Description:
+
+   Creates an archived pill when given:
+
+   page_object:    The page object to check for archived state.
+
+   ========================================================================= #}
+
+{% macro render( page_object ) %}
+
+{% if page_object and page_object.is_archived %}
+	<div class="m-archived-pill">
+	    Archived
+	</div>
+{% endif %}
```

Comment:
I definitely considered that option. I went with this approach (for now, at least) because I thought it might be helpful to keep the code isolated if there are further changes to archiving, and also if we expand archiving beyond the current page types, this pill may need to be added in multiple other locations.

I'm not strongly attached to the approach either, though, so open to changing!

Reply:
I'm going to go ahead and merge as is, but we can revisit this later...